doc/languages-frameworks/cuda.section.md

```diff
@@ -32,3 +32,11 @@ mypkg = let
   }});
 in callPackage { inherit cudaPackages; };
 ```
+
+cudaPackages also provides a default set of cudaCapabilites for cards based on the
```

Comment:
For users reading this documentation, how should they determine what `cudaCapabilities` to set based on the model of GPU they have? Perhaps we could give a mini-explainer or link to a tutorial/docs?

Otherwise this documentation isn't easily actionable for users.